### ISSUE TITLE
Error when schema has a field of type Schema.Types.Buffer with a default of null

### ISSUE THREAD

**Issue by Narretz** [francescov1/mongoose-tsgen#47]
Hi, frist of all, thanks for this package. This is exactly what I wanted  since I don't want to touch my existing schema defs. :)

I think I found a bug related to Buffer in a schema:

Consider this 

```
const homeSchema = new Schema(
  {
    id: String,
    image: {
     type: Schema.Types.Buffer,
     default: null
   }
});

```

For me. this throws an error during gen: `TypeError: Cannot read property 'required' of null` coming from here: https://github.com/francescov1/mongoose-tsgen/blob/854a1f9289bb07107622914bf8e76dbde202d2e5/src/helpers/parser.ts#L468

Now I don't know 100% if Buffer types even can have a default value null?

**Comment by francescov1** [comment]
Hey @Narretz, thanks for submitting the issue!

I investigated and it seems that the way you defined the Buffer type differs slightly from the Mongoose docs, which is why I missed adding a check to handle this. 

I will add support for your format, but if you want to continue using the tool in the meantime you can define your Buffer using the format described [here](https://mongoosejs.com/docs/schematypes.html#buffers). Let me know if this solves the issue! 😊

**Comment by francescov1** [comment]
Fixed in 8.0.2 🚀
